scripts/check-mcp-compliance.py

```
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+"""MCP Compliance Check - Ensure all components use real MCP, no mocks"""
+
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List, Tuple
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)